src/components/img.ts

```diff
@@ -44,7 +44,7 @@ const Img: FC<Props> = ({ image, sizes, className, format }) =>
         styledH('img', {
             src: image.src,
             alt: image.alt.withDefault(''),
-            className: image.launchSlideshow ? 'js-launch-slideshow' : '',
+            className: image.width > 620 ? 'js-launch-slideshow' : '',
```

Comment:
Does this filter out Avatars and Cutouts? We probably also don't want the slideshow for thumbnail images https://www.theguardian.com/media/2020/jun/12/capturing-the-cry-for-change-photographers-on-the-blm-protests

Reply:
Also don't we use a width of `500` for the default image?

Reply:
Seems to filter out unnecessary objects, layout and positioning seems perfect on my screen

Reply:
Layout seems perfect on my end, cutouts seems to filtered and there doesn't seem to be an author

Reply:
This uses the width of the image as specified by the master asset from CAPI. [We specify](https://github.com/guardian/apps-rendering/blob/a01b50bd30993c6a61bf6ef550bfec0a9d1d49f0/src/contributor.ts#L36) a width of `192` for contributor images, so this will filter those out.

Regarding thumbnails, it will include them if their CAPI master asset width is greater than `620`, otherwise it won't. This matches the current behaviour on dotcom and is known to design (cc @paperboyo).

Reply:
Yep. Thumbnails should be included (unless smaller than 620). Contributor images,badges etc from furniture obviously shouldn't (and their masters can be over 620, so they better be excluded based on something else).

Reply:
@paperboyo we don't derive the widths from master assets for contributor images because we don't have them 🙂. We get `bylineLargeImageUrl` from the tag.

Reply:
Perfetto!

[Photo essay](https://www.theguardian.com/media/2020/jun/12/capturing-the-cry-for-change-photographers-on-the-blm-protests) is a dud. And someone made a mistake of creating these circular portraits bigger than 620, so they appear in Lightbox. Nothing we can (or should) bo doing in code to prevent that.